### ISSUE TITLE
Modify handleNick() to fire a custom event

### ISSUE THREAD

**Issue by elazar** [phergie/phergie-irc-plugin-react-nickserv#6]
There are instances where it is desirable for other plugins to be aware of when authentication with NickServ is successful, such as joining channels that require user registration. Modify `handleNick()` to emit a custom event that other plugins can listen for.

**Comment by Renegade334** [comment]
This isn't the right event to listen out for. Instead, the plugin should fire an event when it receives a privmsg or notice with the phrase "identified" (or "now identified"). The bot's nick being changed is more likely to be associated with NickServ implementations that forcibly change a user's nick if they fail to identify to it.

**Comment by elazar** [comment]
Here's the event as Phergie v2 currently receives it from Freenode:

```
[2015-04-08/17:06:27]  irc.freenode.net <- NOTICE Phergie :You are now identified for ^BPhergie^B.
```

@Renegade334 @PSchwisow Is this a change one of you could make against master?

**Comment by PSchwisow** [comment]
@elazar Do you want me to remove the other event I added or leave that in place?

**Comment by elazar** [comment]
@PSchwisow The event is the same, in name at least. The only difference is the circumstances under which it's emitted: rather than in `handleNick()`, it will be emitted when a `PRIVMSG` or `NOTICE` event is received from the NickServ user that contains a configurable phrase with the default being `'now identified'`.

**Comment by elazar** [comment]
@PSchwisow Merged.

@Renegade334 Sufficient?

**Comment by elazar** [comment]
Merged #11.
